build.gradle

```diff
@@ -31,12 +31,11 @@ subprojects {
     afterEvaluate { project ->
         if (project.hasProperty("android")) {
             android {
-                compileSdkVersion 30
-                buildToolsVersion "28.0.3"
+                compileSdkVersion 33
 
                 defaultConfig {
-                    minSdkVersion 19
-                    targetSdkVersion 30
+                    minSdkVersion 21
+                    targetSdkVersion 33
```

Comment:
Do we need to go to 33 here? It means we have tested the code with a 33 device. And that would be Android 13. Does @liliya-soroka  even have a device with Android 13?

I would prefer 30 here (Android 11)

Reply:
android app targets 33, also svalinn use is limited to ens resolution currently